index.html

```diff
@@ -0,0 +1,168 @@
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </div>
+      </div>
+
+      <!-- Navbar with sub-titles link -->
+      <div id="nav-bar">
+        <a href="#" id="logo">COP27<span>2022</span></a>
+        <nav id="menu">
+          <span id="menu-close-btn">×</span>
+          <a href="#" class="home-link">Home</a>
+          <a href="#program">Program</a>
+          <a href="#speakers">Speakers</a>
+          <a href="#sponsors">Sponsors</a>
+          <a href="./about.html">About</a>
+        </nav>
+        <a href="#" id="menu-toggler">
+
+          <!-- Scalable vector graphics for hamburger -->
+          <svg
+            class="w-6 h-6"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path stroke-width="3" d="M4 6h16M4 12h16M4 18h16"></path>
+          </svg>
+        </a>
+      </div>
+    </header>
+
+    <!-- Title welcome page section -->
+    <section id="section1" class="container">
+      <div class="wrapper">
+        <img src="img/bg-bob.png" class="shape-set" alt="" />
+        <h1>World Climate Summit - COP27</h1>
+        <h4>The Investment COP</h4>
+        <h2>"Sustainable Environment"</h2>
+        <p class="description">
+          World Climate Summit -
+          The Investment COP will utilise a decade of experience to bring together
+          high-level representatives from the public and private sectors including
+          businesses, investors, governments, policymakers, innovators, academia and environmental
+          <influencers class=""></influencers>
+        </p>
+        <h3>7th - 18th November, 2022</h3>
+        <p class="location">Sharm El-Sheikh, Egypt</p>
+      </div>
+    </section>
+
+    <!-- Main Program Events -->
+    <section id="program">
+      <h2 class="section-heading">Main Event</h2>
+      <div class="items">
+        <div class="item">
+          <img src="img/icons-1.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Lecture</h3>
+          <p class="item-description">
+            Lecture to address the climate emergency by challenging existing practices and
+            exploring new ideas to drive tangible results
+          </p>
+        </div>
+        <div class="item">
+          <img src="img/icons-5.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Interaction</h3>
+          <p class="item-description">
+            Both events bring together thought-leaders, industry experts and dynamic organisations
+          </p>
+        </div>
+        <div class="item">
+          <img src="img/icons-3.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Awards</h3>
+          <p class="item-description">
+            Awards
+          </p>
+        </div>
+        <div class="item">
+          <img src="img/icons-4.svg" class="item-icon" alt="" />
```

Comment:
- [ ] please kindly use five articles like in the behance design
yours
![Screenshot (542)](https://user-images.githubusercontent.com/65298048/197072845-33e3b1c5-61f1-4d7e-9a24-2b1777d76343.png)
behance
![Screenshot (544)](https://user-images.githubusercontent.com/65298048/197072864-2576d486-1477-4a8e-b3f5-fab907c73a4b.png)

Reply:
Fixed :+1: 
[OPTIONAL]
- [ ] However, You may consider surrounding your card with a white square when it is hovered as in the design :smile: 
- [ ] Also the Participate is not supposed to have that style. As you can see in the template. It is only a simple white link with a text-decoration :smile: